app/presenters/admin/user_presenter.rb

```diff
@@ -0,0 +1,30 @@
+class Admin::UserPresenter < SimpleDelegator
+  include Rails.application.routes.url_helpers
+  include ActionView::Helpers
+  include ActionView::Helpers::TranslationHelper
+  include ActionView::Context
+
+  def initialize(user)
+    @user = user
+    super(@user)
+  end
+
+  def turn_admin_button(is_admin)
+    translation_scope = 'admin.users.index'
+
+    if is_admin
+      btn_text = t('users.admin_off')
+      btn_class = 'btn-warning'
+      confirm_message = t('are you sure you want to remove administration rights from this user', scope: translation_scope)
+    else
+      btn_text = t('users.admin_on')
+      btn_class = 'btn-primary'
+      confirm_message = t('are you sure you want to give administration rights to this user', scope: translation_scope)
+    end
```

Comment:
:thought_balloon: After your change, I'm thinking that this code was better placed in the view

Just adding my opinion here, hahahahaha :stuck_out_tongue_closed_eyes: